types-test.js

```diff
@@ -0,0 +1,9 @@
+/* @flow */
+
+import { load } from './dist/gh-emoji.js';
+
+
+// Flowtype should report this as an error
+const test: Promise<string> = load();
+
+test.then((result) => console.log(test)); //eslint-disable-line
```

Comment:
Same here, instead of turning eslint off maybe we should just remove unused argument?